spec/active_model_errors_spec.rb

```diff
@@ -57,7 +57,7 @@
       end
 
       it 'does not use the original method' do
-        stub(errors)._generate_message
+        RR.stub(errors)._generate_message
```

Comment:
Not so important, but do you know why you had to specify the class here? I thought the `config.mock_with :rr` line would mean you can just write `stub(...`?

Reply:
I replaced this because if I use only stub I get undefined method. And from this https://github.com/rspec/rspec-mocks/pull/233/files I saw that stub was deprecated some time ago

Reply:
Ah I see, I think there's two separate things though:
1. From what I can tell, using the `config.mock_with` option will automatically import `stub` from RR, you shouldn't need the `RR.` class, but 🤷‍♂.
2. I'm not sure if RR is needed, perhaps it's used throughout, but if not, you can use the standard RSpec syntax of `allow(errors).to receive(:_generate_message)`.
I don't think it's particularly important to change it now if the above is working fine though.

Reply:
Thanks for your input. I do not know why it's not working without 'RR.' . I thought about standard RSpec syntax too but since the author started with rr I tried to keep it that way.